index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+  <nav class="navbar">
+    <h4 class="name">Oscar Espinoza</h4>
+    <div class="toggle-btn">
+      <span></span>
+      <span></span>
+      <span></span>
+    </div>
+    <div class="nav-links">
+      <p>Portfolio</p>
+      <p>About</p>
+      <p>Contact</p>
+    </div>
+  </nav>
+  <div class="headline">
+    <h1>Hey there. <br>I'm Oscar</h1>
+    <h2>I'm a software developer</h2>
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</p>
+      <div class="icons">
+        <img src="Media/Icon -GitHub.svg" alt="">
+        <img src="Media/Icon-Linkedin.svg" alt="">
+        <img src="Media/Icon-peace.svg" alt="">
+        <img src="Media/Icon-Twitter.svg" alt="">
+        <img src="Media/Icon-M.svg" alt="">
+      </div>
+      <img src="Media/image.svg" alt="" class="squares">
```

Comment:
Good job adding the `src` and the `alt` attributes to the `img` tag 🥇 

- [ ] Actually, this is a list of different items. In this case, it's apply the best practices of the `ul` tag, and inside them you can create an `li` with an `a` tag for each `img` that you have, so each icon will have his corresponding `href` to the differents social icons.

Kindly make this changes so your different icons can give to the viewer an interactive visit on your social places.